proto/netbookai/spawner/spawner.proto

```diff
@@ -405,3 +408,22 @@ message TagNodeInstanceRequest {
   string nodeGroup = 5;
   map<string, string> labels = 6;
 }
+
+
+message GetCostByTimeRequest {
+  repeated string Ids = 1;
+  string provider = 2;
```

Comment:
can you change the order here, as per other messages, provider will be first field, followed by region and accountName,

Reply:
provider is the secondary field here as cost is requested for Ids also for other cost rpcs, Id is first field

Reply:
in that case moving provider to top make sense right?, in future if ids are to be changed to something else it would make tag:1 useless, IMO, keeping commons at the top maintains consistency with tag numbers.